packages/theme-ui/src/color-modes.js

```diff
@@ -68,4 +68,17 @@ const bodyColor = theme => ({
 
 export const ColorMode = () => <Global styles={bodyColor} />
 
+export const ApplyColorModeFromLocalStorage = () => (
+  <script
+    key="theme-ui-noscript"
```

Comment:
Think I made this typo originally, but... 
```suggestion
    key="theme-ui-no-flash"
```